src/build/arg/mod.rs

```diff
@@ -2497,7 +2497,8 @@ impl<'help> Arg<'help> {
 
     /// Specifies the value of the argument if `arg` has been used at runtime. If `val` is set to
     /// `None`, `arg` only needs to be present. If `val` is set to `"some-val"` then `arg` must be
-    /// present at runtime **and** have the value `val`.
+    /// present at runtime **and** have the value `val`. Setting `default` to `None` removes any
```

Comment:
Instead of adding it here, add a separate note describing the scenario and what happens after the first note below.

Reply:
I moved this line down to the later example.